app/controllers/users_controller.rb

```diff
@@ -118,8 +118,13 @@ def edit
 
   # PUT /facilities/:facility_id/users/:id
   def update
+    previous = @user.price_groups.to_a
```

Comment:
I think we can be a little more explicit in our variable name here: `previous_price_groups`.

Reply:
Sorry, I wrote with the intention to change it once it was running properly and it slip my mind